src/main/reactjs/src/interfaces/authorisation.ts

```diff
@@ -20,6 +20,7 @@ export interface Authorisation
   termEndDate?: Dayjs;
   autDate?: Dayjs;
   translatorId?: number;
+  tempId?: string;
```

Comment:
Tarvisko näitä miettiä? Teoriassa kai mukava, että voidaan pitää yksi tyyppi mutta siellä on nyt sekä translatorId, että tempId kentät, joita tarvitaan vain muutamassa erikoistapauksessa mutta onkos tähän mitään parempaa tapaa?

Reply:
Onkohan tää `tempId` sellainen konsepti josta olisi hyötyä muuallakin? Meillä on olemassa mm. `WithId` ja `WithVersion` interfacet, niistä voisi ehkä saada inspiraatiota.

Reply:
Mjoo, tuuppasin tollasen WithTempId helperin tonne kaveriksi